packages/editor-ui/src/components/Error/NodeErrorView.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div>
 		<div class="error-header">
-			<div class="error-message">{{ $locale.baseText('nodeErrorView.error') + ': ' + getErrorMessage() }}</div>
-			<div class="error-description" v-if="error.description">{{getErrorDescription()}}</div>
+			<div class="error-message">{{ getErrorMessage() }}</div>
+			<div class="error-description" v-if="error.description" v-html="getErrorDescription()"></div>
```

Comment:
Error descriptions have to be able to support links. User can `throw new Error(message)` inside the `jsCode` param but that is the error title, not the description. Since only the BE can set descriptions, I thought XSS risk is not an issue here. Added `sanitizeHtml` to the call just in case.